hie-bios.cabal

```diff
@@ -53,6 +53,7 @@ Library
                         process              >= 1.6.1 && < 1.7,
                         file-embed           >= 0.0.10 && < 0.1,
                         ghc                  >= 8.2.2 && < 8.9,
+                        shellwords           >= 0.1.2 && < 0.2,
```

Comment:
This adds a megaparsec dependency which I want to avoid.

Reply:
Now this PR expects arguments to be newline-delimited. Each argument must be in its own line.

So the only change now is `words res` -> `lines res`.

This will break existing users.